lib/system/ansi_c.nim

```diff
@@ -127,10 +128,16 @@ elif defined(nimRawSetjmp) and not defined(nimStdSetjmp):
     # prone to stack corruption during unwinding, so we disable that by setting
     # it to NULL.
     # More details: https://github.com/status-im/nimbus-eth2/issues/3121
+    when defined(nimHasStyleChecks):
```

Comment:
On other platforms, `c_setjmp` is a `importc`, so the styling doesn't apply (AFAIK)
Here we wrap it, so it's not an `importc`, so styling apply. But if I switched it to `cSetjmp`, like the compiler wanted, it would fail with `styleCheck:usage` since on other platform it expects a `c_setjmp`